website/content/docs/services/discovery/dns-forwarding/enable.mdx

```diff
@@ -0,0 +1,442 @@
+#rev-server=172.16.0.0/12,127.0.0.1#8600
+#rev-server=192.168.0.0/16,127.0.0.1#8600
+#rev-server=224.0.0.0/4,127.0.0.1#8600
+#rev-server=240.0.0.0/4,127.0.0.1#8600
+```
+
+</CodeBlockConfig>
+
+Restart the `dnsmasq` service after creating the configuration.
+
+You can configure additional settings in `dnsmasq` that may be useful in your environments. 
+Refer to [`dnsmasq(8)`](http://www.thekelleys.org.uk/dnsmasq/docs/dnsmasq-man.html) for additional details.
+
+<CodeBlockConfig filename="/etc/dnsmasq.conf">
+
+```plaintext
+# Accept DNS queries only from hosts whose address is on a local subnet.
+#local-service
+
+# Don't poll /etc/resolv.conf for changes.
+#no-poll
+
+# Don't read /etc/resolv.conf. Get upstream servers only from the command
+# line or the dnsmasq configuration file (see the "server" directive below).
+#no-resolv
+
+# Specify IP address(es) of other DNS servers for queries not handled
+# directly by consul. There is normally one 'server' entry set for every
+# 'nameserver' parameter found in '/etc/resolv.conf'. See dnsmasq(8)'s
+# 'server' configuration option for details.
+#server=1.2.3.4
+#server=208.67.222.222
+#server=8.8.8.8
+
+# Set the size of dnsmasq's cache. The default is 150 names. Setting the
+# cache size to zero disables caching.
+#cache-size=65536
+```
+
+</CodeBlockConfig>
+
+## BIND
+
+[BIND](https://www.isc.org/bind/) is a suite of software for interacting with the Domain Name System (DNS). Its most prominent component, `named`, short for name daemon, performs both of the main DNS server roles, acting as an authoritative name server for DNS zones and as a recursive resolver in the network.
+
+<Note>
+
+If your distribution uses systemd, you may need to disable `systemd-resolved` prior to following this example. 
+ 
+</Note>
+
+You can configure the BIND service to send `.consul` domain queries to Consul by:
+- creating a configuration file for `named` with `DNSSEC` disabled
+- creating a zone configuration file to manage the `.consul` domain 
+
+### Named configuration file
+
+Edit the `/etc/named.conf` to configure your BIND instance. Remember to disable `DNSSEC` so that Consul and BIND can communicate. Also add an `include` section to include the zone file you will create in the next step.
+
+The following example shows a BIND configuration with `DNSSEC` disabled.
+
+<CodeBlockConfig filename="/etc/named.conf">
+
+```plaintext
+options {
+  listen-on port 53 { 127.0.0.1; };
+  listen-on-v6 port 53 { ::1; };
+  directory       "/var/named";
+  dump-file       "/var/named/data/cache_dump.db";
+  statistics-file "/var/named/data/named_stats.txt";
+  memstatistics-file "/var/named/data/named_mem_stats.txt";
+  allow-query     { localhost; };
+  recursion yes;
+
+  dnssec-enable no;
+  dnssec-validation no;
+
+  /* Path to ISC DLV key */
+  bindkeys-file "/etc/named.iscdlv.key";
+
+  managed-keys-directory "/var/named/dynamic";
+};
+
+include "/etc/named/consul.conf";
+```
+
+</CodeBlockConfig>
```

Comment:
Wondering if this section should actually be under the first bullet point and the "Zone..." subsection under the second bullet point since configuring BIND is a [multi-step procedure](https://hashicorp.atlassian.net/wiki/x/DICVpg).